build.sbt

```diff
@@ -30,23 +30,19 @@ resolvers += Resolver.jcenterRepo
 
 libraryDependencies += guice
 libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2" % Test
-
-val silhouetteVersion = "5.0.0"
-libraryDependencies ++= Seq(
-  "com.mohiva" %% "play-silhouette" % silhouetteVersion,
-  "com.mohiva" %% "play-silhouette-persistence" % silhouetteVersion,
-  "com.mohiva" %% "play-silhouette-crypto-jca" % silhouetteVersion,
-  "com.mohiva" %% "play-silhouette-password-bcrypt" % silhouetteVersion
-)
```

Comment:
Why did you remove this construct in favor of single dependency entries?
Please revert, if there is no good reason. ;)

Reply:
There was no good reason so I reintroduced this nice construct :)